Sources/PostgresNIO/Connection/PostgresConnection.swift

```diff
@@ -438,6 +438,47 @@ extension PostgresConnection {
         }
     }
 
+    /// Run a simple text-only query on the Postgres server the connection is connected to.
+    /// WARNING: This function is not yet API and is incomplete.
+    /// The return type will change to another stream.
+    ///
+    /// - Parameters:
+    ///   - query: The simple query to run
+    ///   - logger: The `Logger` to log into for the query
+    ///   - file: The file, the query was started in. Used for better error reporting.
+    ///   - line: The line, the query was started in. Used for better error reporting.
+    /// - Returns: A ``PostgresRowSequence`` containing the rows the server sent as the query result.
+    ///            The sequence  be discarded.
+    @discardableResult
+    public func __simpleQuery(
```

Comment:
I'm not sure what will happen to this PR but the name is underscored for now so we can merge this PR as-is and I can then create follow-up PRs.